docs/application/dotnet/guides/maps/maps.md

```diff
@@ -164,34 +164,34 @@ To start using the map service:
 
 2.  Create a `Tizen.Maps.MapService` class instance using the provider name and provider key issued by the map provider:
 
-    ```
+    ```csharp
     var maps = new MapService("MAPS_PROVIDER_NAME", "Your-Maps-Provider-Key");
-    ```
+    
```

Comment:
Missing closing code tag, please udpate it

Reply:
Hi, @piotrekgorny and team, I am doing the final check before merging the PR, I will do the final check today to see if there are any issues, and post that if there are any comments you can fix it tomorrow. 
However, since there are so many files it may not be possible for me to identify any missing issues on short notice. Maybe post merging we can keep a track of the files and raise issues for a set of 10 files.

Reply:
@annie-samsung 
Thank you for your review. We will do our best to fix the issues as soon as possible when you report them.